src/main/java/com/jalasoft/jfc/model/exception/Md5Exception.java

```diff
@@ -0,0 +1,32 @@
+package com.jalasoft.jfc.model.exception;
+
+/**
+ * Class of manage exception of Md5.
+ *
+ * @version 0.1 03 Jan 2020.
+ *
+ * @author Alan Escalera.
+ */
+public class Md5Exception extends Exception {
+
+        // Returns a message error.
```

Comment:
improve this comment to "Returns a error message"

Reply:
ok, it is already fixed.